plugins/modules/one_vm.py

```diff
@@ -1398,7 +1484,8 @@ def main():
         "labels": {"default": [], "type": "list", "elements": "str"},
         "count_labels": {"required": False, "type": "list", "elements": "str"},
         "disk_saveas": {"type": "dict"},
-        "persistent": {"default": False, "type": "bool"}
+        "persistent": {"default": False, "type": "bool"},
+        "updateconf": {"type": "dict"},
```

Comment:
It might make sense to require `instance_ids` when `updateconf` is specified. The `required_by` parameter to `AnsibleModule` should allow to specify that (IIRC `AnsibleModule(..., required_by=dict(instance_ids=['updateconf'])))`).

Reply:
Actually the idea is to support this also for VM creation. The difference is that instead of calling API again, the updateconf dict is merged into the VM template and only then the template is instantiated. There is one example of this in docs. :thinking:

Reply:
Ok. What happens if `instance_ids` is not specified and running VMs are updated? (Or does updating only happen when `instance_ids` is specified? I don't know the module.)

Reply:
I corrected the condition in https://github.com/ansible-collections/community.general/pull/5812/commits/c11167d2af0097b616ce13644c69b500f17183ea, thank you for spotting this. :ok_hand::relieved:

Reply:
Ok, now it makes more sense to me :) (I don't use the module or OpenNebula in general, so I'm doing a lot of guessing and questioning, I'm always glad if it still helps to spot problems ;-) )